packages/documentation/data/menu.ts

```diff
@@ -19,7 +19,6 @@ import {
 	KtNavbar,
 	KtPagination,
 	KtPopover,
-	KtRadio,
```

Comment:
Please mark the 4 relevant tasks as complete in #273. After this PR, we only have `KtInput` to remove, then we can finally release Kotti 3 officially 🚀

Reply:
Also, `KtInput` only appears 5 times in the codebase by now. So, the more challenging issue will be the associated classes.

Reply:
Oh, I just noticed we forgot to add the inline edit to that document. But still, some good progress towards V3.

Reply:
the inline edit is also wip by me now, would be great to take a look when I submit the final changes today so I can also merge the new one :)